tests/conftest.py

```diff
@@ -13,6 +14,22 @@
 
 _T = TypeVar("_T")
 
+CA_CERT_FILE = str(Path(__file__).parent / "resources/ca.pem")
+CLIENT_KEY_FILE = str(Path(__file__).parent / "resources/client-key.pem")
+CLIENT_CERT_FILE = str(Path(__file__).parent / "resources/client.pem")
+SERVER_KEY_FILE = str(Path(__file__).parent / "resources/server-key.pem")
+SERVER_CERT_FILE = str(Path(__file__).parent / "resources/server.pem")
+
+WAIT_TIME_OUT = 10
+
+WAIT_TIME_OUT = 10
+
+CA_CERT_FILE = str(Path(__file__).parent / "resources/ca.pem")
+CLIENT_KEY_FILE = str(Path(__file__).parent / "resources/client-key.pem")
+CLIENT_CERT_FILE = str(Path(__file__).parent / "resources/client.pem")
+SERVER_KEY_FILE = str(Path(__file__).parent / "resources/server-key.pem")
+SERVER_CERT_FILE = str(Path(__file__).parent / "resources/server.pem")
```

Comment:
These seem duplicated

Reply:
will fix.

Reply:
Fixed in [ee02430](https://github.com/opendatahub-io/caikit-nlp-client/pull/16/commits/ee02430be42d16ef76793be70ee9616d23268997)